public/app/features/browse-dashboards/components/BrowseActions/DeleteModal.tsx

```diff
@@ -39,6 +43,16 @@ export const DeleteModal = ({ onConfirm, onDismiss, selectedItems, ...props }: P
           </Text>
           <DescendantCount selectedItems={selectedItems} />
           <Space v={2} />
+          {deleteIsInvalid ? (
+            <Alert
```

Comment:
![image](https://github.com/grafana/grafana/assets/46142/228b9492-5f77-4f9c-9573-ba52b5f9513c)

Alert ends up with font size that's too large, caused by ConfirmModal setting a larger font size on the body with some weirdly named props.

Unsure which is the right solution to this:
 - Remove the font-size in ConfirmModal (I think i prefer this?)
 - Pass an empty `body={<></>}` prop, and pass our content in `description`, which has the body text size.

If we make a change to ConfirmModal we should probably put that in a seperate PR.